Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -160,21 +161,23 @@ class ContentBlockerHelper {
         }
     }
 
-    private func addActiveRulesToTab() {
+    // Function to install or remove TP for a tab
+    @objc func setupTabTrackingProtection() {
```

Comment:
You were right, once the old whitelisting is removed, this function can be called minimally, no more isInstalled state needed as it isn't a concern anymore.